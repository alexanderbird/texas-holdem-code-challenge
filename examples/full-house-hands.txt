6H 8C 6S 8D TC
Lelah 8S 5D
Carter 6D 5H
Starla TS TH
