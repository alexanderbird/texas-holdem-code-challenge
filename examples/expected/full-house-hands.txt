1 Starla Full House 10 8
2 Lelah Full House 8 6
3 Carter Full House 6 8
